File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/explorer/UpdatedBy.java
package app.editors.manager.mvp.models.explorer;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class UpdatedBy implements Cloneable, Serializable {

    @SerializedName("id")
    @Expose
    private String id = "";
    @SerializedName("displayName")
    @Expose
    private String displayName = "";
    @SerializedName("avatarSmall")
    @Expose
    private String avatarSmall = "";
    @SerializedName("profileUrl")
    @Expose
    private String profileUrl = "";

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }

    public String getAvatarSmall() {
        return avatarSmall;
    }

    public void setAvatarSmall(String avatarSmall) {
        this.avatarSmall = avatarSmall;
    }

    public String getProfileUrl() {
        return profileUrl;
    }

    public void setProfileUrl(String profileUrl) {
        this.profileUrl = profileUrl;
    }

    @Override
    public UpdatedBy clone() throws CloneNotSupportedException {
        return (UpdatedBy) super.clone();
    }
}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/explorer/UpdatedBy.kt
package app.documents.core.network.manager.models.explorer

import android.text.Html
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

open class UpdatedBy : Cloneable, Serializable {
    @SerializedName("id")
    @Expose
    var id = ""

    @SerializedName("displayName")
    @Expose
    var displayName = ""

    @SerializedName("avatarSmall")
    @Expose
    var avatarSmall = ""

    @SerializedName("profileUrl")
    @Expose
    var profileUrl = ""

    @SerializedName("hasAvatar")
    @Expose
    var hasAvatar = false

    @SerializedName("isAnonim")
    @Expose
    var isAnonim = false

    @Throws(CloneNotSupportedException::class)
    public override fun clone(): UpdatedBy {
        return super.clone() as UpdatedBy
    }

    val displayNameFromHtml: String
        get() = Html.fromHtml(displayName, Html.FROM_HTML_MODE_LEGACY).toString()
}